libraries/AP_Scripting/applets/pelco_d_antennatracker.lua

```diff
@@ -0,0 +1,114 @@
+--[[
+   Pelco-D control implementation for antennatracker.
+   Implemented by using knowledge from the excellent python implementation in https://gist.github.com/jn0/cc5c78f4a0f447a6fb2e45a5d9efa13d.
+--]]
+
+local SERVO_PAN = 71
+local SERVO_TILT = 72
+local baud = 9600
+
+-- Antennattracker modes
+local MODE_STOP = 1
+local MODE_SCAN = 2
+local MODE_SERVOTEST = 3
+local MODE_GUIDED = 4
+
+gcs:send_text(0, "Starting Pelco-D Control")
+local port = serial:find_serial(0)
```

Comment:
```suggestion
local port = assert(serial:find_serial(0), "Pelco-D: No Scripting Serial Port")
```
To fix the lua CI warning. The assert checks that `port` is not `nil`. Before you call begin on it.

You do have the check on line 21, so you could also move the begin and flow control calls after that. 

With the assert you can remove the check on 21.